staff_features/repositories/repository_create.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Repository Create
+  Scenario: Repository can be created from an admin user
+    Given an administrator user is logged in
+     When the user clicks on 'System'
+      And the user clicks on 'Manage Repositories'
+      And the user clicks on 'Create Repository'
+      And the user fills in 'Repository Short Name'
+      And the user fills in 'Repository Name'
+      And the user clicks on 'Save'
+     Then the 'Repository Created' message is displayed
+  Scenario: Repository cannot be created from an archivist user
+    Given an archivist user is logged in
+     When the user clicks on 'System'
+     Then the 'Manage Repositories' option is not displayed in the dropdown menu
```

Comment:
Something to think about here -- if the name of the menu item for managing repositories changes and the test isn't updated, this test could conceivably pass even if it shouldn't.  Is there a way to make the tests more robust?  Maybe by using string constants that are defined in place so that any tests that rely on a string all get updated if a string changes?  (Also worth thinking about from an internationalization perspective)

Reply:
I totally see the point that this Then step will result in false positives. We need to find another way to assert that the currently logged in user cannot create a repository (perhaps one that is not so tied to the ui?)

Regarding the internationalization perspective, I think we could at some point write some scenarios testing that the internationalization works in a couple of views / behavior scenarios. We could use gherkin data tables to assert the different strings are displayed depending on the selected language.

Generally I have the feeling that trying to write the whole suite so that it works regardless of the currently displayed language would be overkill and probably result in not so easy to maintain step definitions.

Reply:
Ok, yes I agree that might be overkill re internationalization.

Reply:
I updated the 'Then' clause to minimize its reliance on the UI and avoid false positive outcomes.